idt_config/linux/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/zsh
+SOURCE_PATH=$1
+IDE_PATH=<PATH TO IDE e.g. /Applications/MCUXpressoIDE_x.y.z>
+TOOLCHAIN_PATH=$IDE_PATH/ide/tools/bin
+IDE=$IDE_PATH/ide/MCUXpressoIDE.app/Contents/MacOS/mcuxpressoide
+rm -r $SOURCE_PATH/.metadata
+$IDE -nosplash -application org.eclipse.cdt.managedbuilder.core.headlessbuild -data $SOURCE_PATH -verbose -import "$SOURCE_PATH/projects/evkmimxrt1060/bootloader" && \
```

Comment:
Eclipse sometimes fails to build if "import" and "build" are done in separate steps. Combining into one step would be better.